core/src/test/scala/org/apache/spark/SparkContextSuite.scala

```diff
@@ -165,6 +165,17 @@ class SparkContextSuite extends SparkFunSuite with LocalSparkContext with Eventu
     }
   }
 
+  test("add hdfs jar files not exists") {
+    try {
+      val jarPath = "hdfs:///no/path/to/TestUDTF.jar"
+      sc = new SparkContext(new SparkConf().setAppName("test").setMaster("local"))
+      sc.addJar(jarPath)
+      assert(sc.listJars().forall(!_..contains("TestUDTF.jar")))
```

Comment:
I think you have an extra period here

Reply:
== 
Before commit code,  accidentally hit the keyboard, have change it .